platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/topology/StaticTopology.java

```diff
@@ -17,78 +17,64 @@
 package com.swirlds.platform.network.topology;
 
 import com.swirlds.common.platform.NodeId;
+import com.swirlds.platform.network.PeerInfo;
 import com.swirlds.platform.network.RandomGraph;
-import com.swirlds.platform.system.address.AddressBook;
 import edu.umd.cs.findbugs.annotations.NonNull;
-import java.util.Arrays;
+import java.util.HashMap;
 import java.util.List;
-import java.util.Objects;
+import java.util.Map;
 import java.util.Random;
-import java.util.function.Predicate;
+import java.util.Set;
 
 /**
  * A bidirectional topology that never changes.
  */
 public class StaticTopology implements NetworkTopology {
     private static final long SEED = 0;
 
-    private final NodeId selfId;
+    /** nodes are mapped so lookups are efficient. **/
+    private Map<NodeId, Long> peerNodeToIdMap = new HashMap<>();
```

Comment:
The name of the variable is more appropriately named as `nodeIdToIndexMap` assuming the intent is to replace the address book lookup for index.

Reply:
updated.